lib/ansible/cli/galaxy.py

```diff
@@ -1451,13 +1458,18 @@ def execute_list_collection(self, artifacts_manager=None):
                     display.vvv("No collections found at {0}".format(collection_path))
                     continue
 
-                # Display header
-                fqcn_width, version_width = _get_collection_widths(collections)
-                _display_header(collection_path, 'Collection', 'Version', fqcn_width, version_width)
+                if output_format in ('yaml', 'json'):
```

Comment:
Semantically, it's better to use a set here:
```suggestion
                if output_format in {'yaml', 'json'}:
```

Reply:
Done.